bazel/build-version.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""
```

Comment:
Make sure to add the Verible authors Apache license header (like in the previous bash script)